pkg/controller/controller_binding.go

```diff
@@ -34,6 +34,9 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
+// bindingControllerKind contains the schema.GroupVersionKind for this controller type.
+var bindingControllerKind = v1alpha1.SchemeGroupVersion.WithKind("Binding")
```

Comment:
I'm not sure if this is the right place for this line, but I have no immediate gut feeling about where it should go - let's leave it here with an eye on whether this proves out to be the right place.